packages/core/src/button/button.element.ts

```diff
@@ -155,12 +155,12 @@ export class CdsButton extends CdsBaseButton {
         this.disableButton();
         return;
       default:
-        this.enableButton();
+        this.disabled ? this.disableButton() : this.enableButton();
     }
   }
 
   private disableButton() {
-    this.style.width = getElementWidth(this);
+    this.style.removeProperty('width');
```

Comment:
What is this change here intended for?

Reply:
When switching between button with loading icon and button with text, the button width will be changed. This line ensures the button width looks normal, but with the guard, I think this is not needed anymore. Great suggestions @mathisscott !

Reply:
Thanks again for the contribution @wghglory. FYI I moved your commit over to the new PR: https://github.com/vmware/clarity/pull/6048/commits

Reply:
@hippee-lee Great! Thank you for letting me know and making this change!